integration/utils/contLoadGenerator.go

```diff
@@ -380,6 +455,125 @@ func (lg *ContLoadGenerator) sendAddKeyTx(workerID int) {
 	<-ch
 }
 
+func (lg *ContLoadGenerator) addKeysToProposerAccount(proposerPayerAccount *flowAccount) error {
+	if proposerPayerAccount == nil {
+		return errors.New("proposerPayerAccount is nil")
+	}
+
+	addKeysToPayerTx, err := lg.createAddKeyTx(*lg.accounts[0].address, lg.constExecParam.PayerKeyCount)
+	if err != nil {
+		lg.log.Error().Msg("failed to create add-key transaction for const-exec")
+		return err
+	}
+	addKeysToPayerTx.SetReferenceBlockID(lg.follower.BlockID()).
+		SetProposalKey(*lg.accounts[0].address, 0, lg.accounts[0].seqNumber).
+		SetPayer(*lg.accounts[0].address)
+
+	lg.log.Info().Msg("signing the add-key transaction for const-exec")
+	err = lg.accounts[0].signTx(addKeysToPayerTx, 0)
+	if err != nil {
+		lg.log.Error().Err(err).Msg("error signing the add-key transaction for const-exec")
+		return err
+	}
+
+	lg.log.Info().Msg("issuing the add-key transaction for const-exec")
+	ch, err := lg.sendTx(0, addKeysToPayerTx)
+	if err != nil {
+		return err
+	}
+	<-ch
+
+	lg.log.Info().Msg("the add-key transaction for const-exec is done")
+	return nil
+}
+
+func (lg *ContLoadGenerator) getConstExecCostTxSizeWithoutComment() uint {
+	emptyTxBody := flow.TransactionBody{}
+	emptyTxBodySize := unsafe.Sizeof(emptyTxBody)
+	authorizerListStr := generateAuthAccountParamList(lg.constExecParam.AuthAccountNum)
+	oneSignatureLen := uint(flow.AddressLength) + flowCrypto.SignatureLenECDSAP256 + 8 /* keyId is uint64 */
+
+	return lg.constExecParam.ArgSizeInByte +
+		uint(emptyTxBodySize) +
+		uint(len(authorizerListStr)) +
+		uint(len(constExecTransactionTemplate)) +
+		lg.constExecParam.AuthAccountNum*oneSignatureLen +
+		lg.constExecParam.PayerKeyCount*oneSignatureLen
+}
+
+func (lg *ContLoadGenerator) sendConstExecCostTx(workerID int) {
+	log := lg.log.With().Int("workerID", workerID).Logger()
+
+	txSizeWithoutComment := lg.getConstExecCostTxSizeWithoutComment()
+	if lg.constExecParam.MaxTxSizeInByte < txSizeWithoutComment {
+		log.Error().Msg(fmt.Sprintf("current tx size(%d) without comment "+
+			"is larger than max tx size configured(%d)",
+			txSizeWithoutComment, lg.constExecParam.MaxTxSizeInByte))
+		return
+	}
+
+	commentSizeInByte := lg.constExecParam.MaxTxSizeInByte - txSizeWithoutComment
+
+	log.Trace().Uint("Max Tx Size", lg.constExecParam.MaxTxSizeInByte).
+		Uint("Tx Arg Size", lg.constExecParam.ArgSizeInByte).
+		Uint("Num of Authorizers", lg.constExecParam.AuthAccountNum).
+		Uint("Num of payer keys", lg.constExecParam.PayerKeyCount).
+		Uint("Script comment length", commentSizeInByte).
+		Msg("Generating one const-exec transaction")
+
+	// Note: comment string are randomly generated so each generated tx payload will differ
+	txScript := ConstExecCostTransaction(lg.constExecParam.AuthAccountNum, commentSizeInByte)
+
+	tx := flowsdk.NewTransaction().
+		SetReferenceBlockID(lg.follower.BlockID()).
+		SetScript(txScript).
+		SetGasLimit(9999).
```

Comment:
We could set a lower gas limit to emphasise that these transactions are empty. (some gas will still be used I think though so maybe 10 is enough)

Reply:
done.